modules/services/nsi.js

```diff
@@ -349,6 +349,7 @@ function gatherNames(tags) {
   }
 
   function isNamelike(osmkey, which) {
+    if (osmkey === 'old_name') return false;
```

Comment:
Perhaps a small detail but it would be cleaner to include this to 'notNames' exception list

Reply:
I thought about that but `notNames` is already a pretty gnarly regex and I didn't want to make it worse.